### DISCUSSION TITLE
[Connector Request] Burner or In Memory or Local wallet

### DISCUSSION THREAD

**Discussion by vrde** [wevm/wagmi#4254] [Connector Request]
### What **novel use-case** does the Connector provide?

The connector would allow:

- easy/fast onboarding: users can interact with the dapp without a wallet
- passkey support: the user's wallet can be restored when a passkey is supplied
- testing: sometimes it's easier and faster to use a local wallet and swap private keys to demo something or for quick prototyping

### Are the Connector's integrations production-ready and generally available?

I guess so?

### Are you committed to actively maintaining the Connector?

- [X] Yes, my team is or I am committed to actively maintaining the Connector.

**Comment by tmm** [comment]
Haven't tried it out, but the `burner-connector` package by the Scaffold ETH team might be worth checking out

https://github.com/scaffold-eth/burner-connector
